Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -276,8 +276,10 @@ MatrixWorkspace_uptr MonteCarloAbsorption::doSimulation(
       } else {
         // elastic case already initialized
       }
+      std::string debugString;
       std::tie(outY[j], std::ignore) =
-          strategy.calculate(rng, detPos, lambdaIn, lambdaOut);
+          strategy.calculate(rng, detPos, lambdaIn, lambdaOut, debugString);
+      g_log.debug(debugString);
```

Comment:
Ditto about moving this to 
`g_log.debug(strategy.getDebugString());`

Reply:
Done